### PR TITLE
Add sad path test for user registration with duplicate email

### LINKED ISSUE #53: SadPaths
Friendship sad path
Login Sad path

### PR DESCRIPTION
**Pull request by brian-greeson**
# Sad paths
Closes: #53 
 - Uncommented Visitor test where new user has duplicate email as another user
  - Modified user_controller to render sad path

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -8,9 +8,9 @@ def new
...
 
   def create
-    user = User.create(user_params)
-    if user.save
-      session[:user_id] = user.id
+    @user = User.new(user_params)
+    if @user.save
+      session[:user_id] = @user.id
       redirect_to dashboard_path
     else
```
Comment (StephanieFriend) [inline app/controllers/users_controller.rb:15]: Great thinking!

File: spec/features/vistors/visitor_can_register_spec.rb [hunk 2]
```diff
@@ -34,28 +34,28 @@
...
+    fill_in 'user[email]', with: email
+    fill_in 'user[first_name]', with: first_name
+    fill_in 'user[last_name]', with: last_name
+    fill_in 'user[password]', with: password
+    fill_in 'user[password_confirmation]', with: password
+  
+    click_on'Create Account'
+  
+    expect(page).to have_content('Username already exists')
+
```
Comment (StephanieFriend) [inline spec/features/vistors/visitor_can_register_spec.rb:59]: Nice!

**Review by StephanieFriend — APPROVED** [review]
Nice job with the sad path testing!
